assets/js/factory/createButton.js

```diff
@@ -1,8 +1,8 @@
 function createButton(classToAdd, ICON, ariaLabel) {
-    const button = document.createElement("button");
-    button.classList.add(classToAdd);
-    button.innerHTML = ICON;
-    button.setAttribute("aria-label", ariaLabel);
-    return button;
-  }
-export default createButton
\ No newline at end of file
+  const button = document.createElement("button");
+  button.classList.add(classToAdd);
+  button.innerHTML = ICON;
+  button.setAttribute("aria-label", ariaLabel);
+  return button;
+}
+export default createButton;
```

Comment:
see https://github.com/rsa-wtag/Todo-App-with-JS/pull/4/files#r1161467720

```suggestion
}

export default createButton;
```